plugins/wysiwygarea/plugin.js

```diff
@@ -93,8 +93,43 @@
 						return;
 					}
 
-					editor.editable( new framedWysiwyg( editor, iframe.$.contentWindow.document.body ) );
+					editor.editable( new framedWysiwyg( editor, iframe.getFrameDocument().getBody() ) );
 					editor.setData( editor.getData( 1 ), callback );
+
+					shouldRecreateEditable( iframe, false );
+
+					editor.on( 'mode', attachIframeReloader, { iframe: iframe, editor: editor, callback: callback } );
+				}
+
+				// Additional permanent onLoad (#4462).
+				function attachIframeReloader( evt ) {
+					evt && evt.removeListener();
+					var iframe = this.iframe,
+						editor = this.editor,
+						callback = this.callback;
+
+					iframe.on( 'load', function() {
+						if ( shouldRecreateEditable( iframe ) ) {
+							shouldRecreateEditable( iframe, false );
+							return;
+						}
+
+						var cacheData = editor.getData( false );
+
+						// Remove current editable, but preserve iframe.
+						editor.editable().saveIframe = true;
+						editor.editable( null );
+
+						var newEditable = new framedWysiwyg( editor, iframe.getFrameDocument().getBody() );
```

Comment:
According to [style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#2222-one-variable-per-scope) we should have here:

```suggestion
						var cacheData = editor.getData( false ),
							newEditable;

						// Remove current editable, but preserve iframe.
						editor.editable().saveIframe = true;
						editor.editable( null );

						newEditable = new framedWysiwyg( editor, iframe.getFrameDocument().getBody() );
```

I know it's an additional line of code, but we promote readability and consistency.